packages/valko-ui-docs/src/pages/docs/forms/CheckboxPage.vue

```diff
@@ -1,3 +1,390 @@
+<script setup lang="ts">
+import { ref, watch } from 'vue'
+
+const cbVariant = ref('primary')
+const cbType = ref('filled')
+const cbSize = ref('md')
+const cbDisabled = ref(false)
+const cbFlat = ref(false)
+const cbRounded = ref(false)
+const checked = ref<boolean | null>(false)
+const cbErrorText = ref(false)
+const cbLabelPosition = ref(false)
+const cbIndeterminate = ref(false)
+const cbReadOnly = ref(false)
+
+/* 
+* This two watch functions control the indeterminate checkbox
+* in the documentation page so it is sincronized 
+* with the status of the checkbox in the sandbox
+*/
+watch(checked, () => {
+  if (checked.value !== null) cbIndeterminate.value = false
+})
+watch(cbIndeterminate, () => {
+  if (cbIndeterminate.value) checked.value = null
+  else checked.value = true
+})
+watch([checked, cbReadOnly], ([newChecked, newReadOnly]) => {
+  if (newReadOnly) {
+    if (newChecked !== true) {
+      checked.value = true
+    }
+  }
+})
+</script>
+
 <template>
-  CheckboxPage
-</template>
\ No newline at end of file
+  <div class="container">
+    <h1>Checkbox</h1>
+    <hr>
+    <section class="w-full">
+      The checkbox component provides users the ability to choose between two distinct values. These are very similar to a switch and can be used in complex forms and checklists
+    </section>
+
+    <section class="w-full">
+      <h2>Playground</h2>
+      <hr>
+      <div class="flex">
+        <div class="w-1/2 flex justify-center items-center border border-light-4 dark:border-dark-1 rounded">
+          <vk-checkbox
+            :variant="cbVariant"
+            :type="cbType"
+            :size="cbSize"
+            :error-text="cbErrorText"
+            :disabled="cbDisabled"
+            :flat="cbFlat"
+            :rounded="cbRounded"
+            :label-position="cbLabelPosition"
+            :read-only="cbReadOnly"
+            :indeterminate="cbIndeterminate"
+            v-model="checked"
+            label="Checkbox"
+          >
+            Primary
+          </vk-checkbox>
```

Comment:
Component doesn't have a slot so the 'Primary' word should be deleted
```suggestion
          <vk-checkbox
            :variant="cbVariant"
            :type="cbType"
            :size="cbSize"
            :error-text="cbErrorText"
            :disabled="cbDisabled"
            :flat="cbFlat"
            :rounded="cbRounded"
            :label-position="cbLabelPosition"
            :read-only="cbReadOnly"
            :indeterminate="cbIndeterminate"
            v-model="checked"
            label="Checkbox"
          />
```